src/Functions/ObjectTrait.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Copyright 2016 Intacct Corporation.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. You may obtain a copy
+ *  of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+namespace Intacct\Functions;
+
+
+use Doctrine\Instantiator\Exception\InvalidArgumentException;
```

Comment:
Was this addressed correctly in the latest pull request?

From: James McGonegal [mailto:notifications@github.com]
Sent: Monday, May 9, 2016 2:25 PM
To: Intacct/intacct-sdk-php <intacct-sdk-php@noreply.github.com>
Cc: Ho, Coleen <cho@intacct.com>; Author <author@noreply.github.com>
Subject: Re: [Intacct/intacct-sdk-php] Adding more unit tests for read methods (#26)


In src/Functions/ObjectTrait.php<https://github.com/Intacct/intacct-sdk-php/pull/26#discussion_r62575795>:

> + *  use this file except in compliance with the License. You may obtain a copy

> + *  of the License at

> + *

> + * http://www.apache.org/licenses/LICENSE-2.0

> + *

> + * or in the "LICENSE" file accompanying this file. This file is distributed on

> + * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either

> + * express or implied. See the License for the specific language governing

> + * permissions and limitations under the License.

> + *

> + */

> +

> +namespace Intacct\Functions;

> +

> +

> +use Doctrine\Instantiator\Exception\InvalidArgumentException;

Autofill got you -- change to just a "use InvalidArgumentException;"

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/Intacct/intacct-sdk-php/pull/26/files/3d6ef5bf5916164821c61bd517a2b443dc0ccb7e#r62575795>